contribution/documentation.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: sideMenu
+title: "Documentation"
+description: ""
+group: nav-contrib
+menu: nav-contrib-front
```

Comment:
You might want to remove this property otherwise you will have a side menu with front-end Guidelines content :)